core/src/i18n/en.yaml

```
@@ -154,9 +154,33 @@ service-management:
 services:
   title: Services
 settings:
-  confirmations: Confirmations
-  dontConfirmDelete: Delete without confirmation
+  clusters:
+    confirmations: Confirmations
+    description: Cluster interaction
+    dontConfirmDelete: Delete without confirmation
+    showHiddenNamespaces: Show Hidden Namespaces
+    title: Clusters
+  interface:
+    title: Interface
+    description: Language, theme
+    themes:
+      hcb:
+        title: High-Contrast Black
+        description: Optimized contrast and accessibility for bright environments
+      hcw:
+        title: High-Contrast White
+        description: Optimized contrast and accessibility for dark environments
+      dark:
+        title: Dark
+        description: Use in dimmed environments
+      light:
+        title: Light
+        description: Use in a default office environment
```

```suggestion
        description: Use in default office environments
```

Consistency with Dark theme description